packages/amplify_core/android/src/test/kotlin/com/amazonaws/amplify/amplify_core/AtomicResultTest.kt

```diff
@@ -15,10 +15,8 @@
 package com.amazonaws.amplify.amplify_core
 
 import io.flutter.plugin.common.MethodChannel
-import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.ExperimentalCoroutinesApi
-import kotlinx.coroutines.Job
-import kotlinx.coroutines.launch
+import kotlinx.coroutines.*
```

Comment:
I thought the codebase is against wildcards importing.

Reply:
Hmm I typically just let AS figure it out. Is there a setting to disable it?